packages/stjude-cloud-theme-react/src/Navbar.js

```diff
@@ -3,11 +3,15 @@ import PropTypes from "prop-types";
 import { Navbar as BSNavbar, Nav, Dropdown, Button } from "react-bootstrap";
 
 let Link;
-try {
-  Link = require("react-router-dom").Link;
-} catch (err) {
-  Link = "a";
-}
+Link = "a";
+// Invariant failed: You should not use <Link> outside a <Router>
+// ^^ This was the error we saw after upgrading to the latest React.
+// 
+//try {
+  //Link = require("react-router-dom").Link;
+//} catch (err) {
+  //Link = "a";
+//}
```

Comment:
Removing this will break in-app linking to pages. E.g., on Platform, the links to the login page would cause a full page reload to `/login`.

As I recall, the invariant error is affected only within this app when viewing/compiling the components in Storybook because of the `react-router-dom` _is_ available. However, when used elsewhere, it may or may not be available.

Reply:
Thanks, this is the piece I wanted to connect about as I realize it will cause a full-page reload. The issue was cropping up when trying to use the component on Platform connecting to the React package locally, and I did not connect it to Storybook but that makes sense. I'll give it a shot.